Emby.Server.Implementations/ScheduledTasks/Tasks/DeleteCacheFileTask.cs

```diff
@@ -25,6 +26,7 @@ public class DeleteCacheFileTask : IScheduledTask, IConfigurableScheduledTask
         private readonly ILogger _logger;
 
         private readonly IFileSystem _fileSystem;
+        private readonly ILocalizationManager _localization;
```

Comment:
This is never initialized

Reply:
Why @mark-monteiro ? Newbie in C#, I don't understand exactly why

Reply:
@mark-monteiro I think that I know what you mean, give me a second